physical_properties/pKa/experimental_data/pKa_experimental_values.csv

```diff
@@ -0,0 +1,25 @@
+Molecule ID,pKa1 mean,pKa1 SEM,pKa2 mean,pKa2 SEM,pKa3 mean,pKa3 SEM,Assay Type,Experimental Molecule ID
+SM01,9.53,0.01,nan,nan,nan,nan,UV-metric pKa,M01
+SM02,5.03,0.01,nan,nan,nan,nan,"UV-metric psKa, Yasuda-Shedlovsky Extrapolation",M02
```

Comment:
Why are there so many nan? does that mean very few molecules actually have replicates? Could you add a comment to the top of the file explaining what is contained here?

Reply:
Some molecules have one experimental pKa, others have two or three macroscopic pKas. I reported NaN for pKa2 and pKa3 for molecules with only one pKa. If all these NaN values are distracting, I can just leave those cells empty.

Reply:
Oh, I thought this was replicate measurements not separate measurements. That makes more sense. I may have miss read something in the README.